src/components/InformationBoxComponent/InformationBoxComponent.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./InformationBoxComponentStyles";
```

Comment:
this gives error upon compiling
should be `import styles from "./InformationBox.module.css"`

Reply:
this left..